src/xUnitToJUnit/xUnitToJUnit.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0;net7.0</TargetFrameworks>
```

Comment:
Can you please modify the [README](https://github.com/gabrielweyer/xunit-to-junit/blob/6d70b8b7ad9e9b305f105450bdee75a22788a9d0/README.md#L11) as well? You'll need to remove `net5.0` and add `net7.0`.